repositories/src/main/scala/cromwell/pipeline/datastorage/dao/entry/ProjectEntry.scala

```diff
@@ -2,7 +2,9 @@ package cromwell.pipeline.datastorage.dao
 
 import cromwell.pipeline.datastorage.Profile
 import cromwell.pipeline.datastorage.dao.entry.UserEntry
-import cromwell.pipeline.datastorage.dto.{ Project, ProjectId, UserId }
+import cromwell.pipeline.datastorage.dto.{Project, ProjectId, UserId}
```

Comment:
Problem with formatter again. Use Ctrl+Shift+O to Optimize import first and then Ctrl+Shift+L to Reformat code

Reply:
I fixed that and push, but I don't know why, but my commit not displayed here.